### PR TITLE
feat(1951): Pass token to executor functions

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

We need token to make api calls to queue service, currently it is not passed but generated in executor

## Objective

This PR will pass token as a param to executor which will be used to make api calls

## References

https://github.com/screwdriver-cd/screwdriver/issues/1951

## License

 confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: lib/build.js [hunk 1]
```diff
@@ -458,8 +472,7 @@ class BuildModel extends BaseModel {
                         eventId: this.eventId,
                         container: this.container,
                         tokenGen: this[tokenGen],
-                        token: this[tokenGen](this.id, tokenGenConfig,
-                            pipeline.scmContext, TEMPORAL_JWT_TIMEOUT)
+                        token: getToken.call(this, job, pipeline)
```
Comment (tkyi) [inline lib/build.js]: should this just be 
`getToken(this, job, pipeline)`?

File: lib/build.js [hunk 2]
```diff
@@ -209,6 +209,31 @@ function getStatusConfig({ metadata }) {
     return statusConfigs;
 }
 
+/**
+ * gets the token
+ * @param {Object} this 
+ * @param {Object} job 
+ * @param {Object} pipeline
```
Comment (tkyi) [inline lib/build.js]: missing descriptions and return type

**Review by jithine — COMMENTED** [review]
can you update `eslint-config-screwdriver` to 5@

**Comment by pritamstyz4ever** [comment]
> can you update `eslint-config-screwdriver` to 5@

Done, updated linter version

**Review by jithine — APPROVED** [review]

**Pull request merged**